src/attenuation.ts

```diff
@@ -123,6 +124,16 @@ export function capabilities<A>(
             // pass through capability escalations from parents
             if (isCapabilityEscalation(parsedParentCap)) {
               yield parsedParentCap
+
+            } else if (
+              capability.isCapability(parsedChildCap.capability) &&
+              (
+                parsedChildCap.capability.with.scheme.toLowerCase() === "as" ||
+                parsedChildCap.capability.with.scheme.toLowerCase() === "my"
```

Comment:
I don't think we need to lowercase the scheme here. AFAIK URIs are case sensitive, and the spec specifies them as lower case, so let's just compare them directly.

Reply:
I would assume that URIs with the schemes `my` and `MY` are the same (just like our `resourcePointer.isEqual()` function does). For example, `http://` and `HTTP://` means the same thing. And, `toLowerCase()` isn't exactly a performance hit, so I say we keep it.